src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -77,6 +77,8 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
     seatedAtTable && venue?.id
   );
 
+  const isReactionsAudioDisabled = !venue.isReactionsMuted;
```

Comment:
[NOTE]
Not really providing any more clarity than `isAudioDisabled={!venue.isReactionsMuted}`, just adds few lines.
It is fine either way.